infrastructure/network/addressmanager/network.go

```diff
@@ -77,6 +78,37 @@ var (
 	heNet = ipNet("2001:470::", 32, 128)
 )
 
+const (
+	// TriedBucketCount is the number of buckets we split tried
+	// addresses over.
+	TriedBucketCount = 64
```

Comment:
It seems like all these consts are now redundant (after KnownAddress is deleted)